app/services/rbac.rb

```diff
@@ -9,10 +9,11 @@ class Rbac
   OPTS = { limit: 1000, auth_names: '' }.freeze
   ANY = '*'
 
+  INVENTORY_UNGROUPED_ENTRIES = [].freeze
+  INVENTORY_HOSTS_READ = 'inventory:hosts:read'
```

Comment:
What about `inventory:hosts:*` 🤔 isn't that a valid one?

Reply:
Covered in test `'global access to hosts'`.